### PR TITLE
feat(ICP-Rosetta): FI-1408: add symbol check

### PR DESCRIPTION
**Pull request by NikolasHai**
This MR proposes the following changes: 

1. Add a check for the passed icrc1 symbol

### CODE REVIEW

File: rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs [hunk 1]
```diff
@@ -134,12 +132,19 @@ pub async fn start_rosetta(rosetta_bin: &Path, arguments: RosettaOptions) -> Ros
         sleep(Duration::from_millis(100)).await;
         tries_left -= 1;
     }
-    let port = maybe_port.expect("Error reading port from port file");
+    match maybe_port {
```
Comment (maciejdfinity) [inline rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs:135]: Are the changes related to port file needed for the symbol check? If now, could we move them to a separate PR?

Comment (mbjorkqvist) [reply rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs:135]: I had the same question, otherwise the changes LGTM, thanks @NikolasHai!

Comment (NikolasHai) [reply rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs:135]: I do not really make a change to the port file. I just need the output of the error in order to be able to check the outputs content against the expected error message, as I do in the system test changes.

Comment (mbjorkqvist) [reply rs/rosetta-api/icrc1/rosetta/runner/src/lib.rs:135]: Ah, I see now, thanks for clarifying!

**Review by mbjorkqvist — APPROVED** [review]
